### PR TITLE
Anomali ThreatStream change DBot verdict from Benign to Unknown for Low Confidence Indicators

### PR DESCRIPTION
**Pull request by zdrouse**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description

Indicators found in Anomali that are below Low Confidence thresholds should be created as Unknown and not Benign.  

Anomali ThreatStream documentation regarding Confidence
https://ui.threatstream.com/optic-doc/Content/Features/threat_model/Observables/details_indicator.htm

Confidence - Confidence indicates the certainty that an observable exhibits or is connected to malicious behavior.

If Anomali has indicators with low Confidence, that doesn't mean the indicator is Benign/Safe.  It means Anomali is unsure that the indicator is Malicious and as such the more appropriate verdict in XSOAR should be Unknown.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
Hi @zdrouse ,
Thank you for your contribution!
can you please add screenshot from Anomali ThreatStream documentation about this?

**Comment by zdrouse** [comment]
> Hi @zdrouse , Thank you for your contribution! can you please add screenshot from Anomali ThreatStream documentation about this?

Attached. 

Screenshot 2023-11-24 at 9 53 18 AM

**Comment by zdrouse** [comment]
For example, if the certainty of an observable (indicator) is 10 (out of 100) that means there is low confidence that it exhibits malicious behavior.  It does not mean it is safe/benign.

**Review by MLainer1 — APPROVED** [review]
Nice! 
:tim:

**Review by adi88d — APPROVED** [review]

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.yml [hunk 1]
```diff
@@ -72,6 +72,15 @@ configuration:
   required: true
   type: 15
   section: Collect
+- defaultvalue: Benign
+  display: Default DBOT score for indicators with low Confidence
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.yml:76]:
```suggestion
  display: Default DBOT score for indicators with low confidence
```

File: Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ If you are upgrading from a previous version of this integration, see [Breaking
     | Use system proxy settings |  | False |
     | Create relationships | Create relationships between indicators as part of enrichment. | False |
     | Remote API | Gather additional information about the threat model from remote APIs. | False |
+    | Default DBOT score for indicators with low Confidence |  | False |
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/README.md:28]:
```suggestion
    | Default DBOT score for indicators with low confidence |  | False |
```

File: Packs/Anomali_ThreatStream/ReleaseNotes/2_2_6.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Anomali ThreatStream v3
+- Added a new parameter **Default DBOT score for indicators with low Confidence** - toggle between `Unknown` and `Benign`.
```
Comment (ShirleyDenkberg) [inline Packs/Anomali_ThreatStream/ReleaseNotes/2_2_6.md:5]:
```suggestion
- Added a new parameter **Default DBOT score for indicators with low Confidence**. Toggle between `Unknown` and `Benign`.
```

**Comment by ShirleyDenkberg** [comment]
@adi88d @MLainer1 @Ni-Knight Doc review completed.

**Pull request merged**